lib/config/error-codes/security.json

```diff
@@ -176,6 +176,18 @@
           "message": "Missing or malformed \"test\" attribute  for role %s (%s) : %s",
           "class": "BadRequestError",
           "deprecated": true
+        },
+        "unknown_controller": {
```

Comment:
This error should not exists since we cannot tell if the user misspell a native controller name or just wanted to set a plugin controller right